client/src/components/Header/Header.js

```diff
@@ -197,7 +197,7 @@ export const Header = ({ isDark = false, loaded }) => {
   const [open, setOpen] = useState(false)
 
   const size = useWindowSize()
-  const isMobile = size.width < 622
+  const narrowScreen = size.width < 1024
```

Comment:
Would be nice to move this to a constant somewhere. Do we have other magic numbers in the code?

Reply:
Seems like this is the only one. I'll remove and apply condition via tailwind as @momentiris suggests below.